src/client.js

```diff
@@ -107,13 +104,23 @@ class ShotgunApiClient {
 		return body;
 	}
 
+	async tokenExpired(token) {
```

Comment:
All operations in this function are synchronous, so this is not needed.
```suggestion
	tokenExpired(token) {
```